README.md

```diff
@@ -5,7 +5,18 @@ Exercism Exercises in Elm
 
 ## Setup
 
-Detailed installation instructions can be found at [http://elm-lang.org/install](http://elm-lang.org/install).
+- Detailed installation instructions for elm can be found at [http://elm-lang.org/install](http://elm-lang.org/install).
+
+- Install `elm-test` for the command line with `npm install elm-test`.
+
+### Running Tests in an Exercise Directory
+Inside an exercise directory, Elm must be configured manually.
+
+- Install `laszlopandy/elm-console` and `deadfoxygrandpa/elm-test`.
+  - `elm-package install -y laszlopandy/elm-console`
+  - `elm-package install -y deadfoxygrandpa/elm-test`
+- You should then be able to run tests with `elm-test ExerciseModuleNameTest.elm`
+
```

Comment:
Yeah, I guess that would go in https://github.com/exercism/xelm/blob/master/docs/TESTS.md ?

I was trying to update that a bit more, but now I'm torn. Since we require elm-test and elm-console, users will need to reinstall both of them in every exercise folder. The least gross option I can think of is to include an elm-package.json file in each exercise directory so that the first run of elm-test will automatically install things for them. Is that kosher @kytrinyx @bewatts ?